src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -88,9 +88,20 @@ export class TelemetryComponent extends CdForm implements OnInit {
     this.configForm = this.formBuilder.group(controlsConfig);
   }
 
+  private replacer(key: string, value: any) {
+    if ((key === 'ranges' || key === 'values') && Array.isArray(value)) {
+      const x = [];
+      for (let i = 0; i < value.length; i++) {
+        x.push(JSON.stringify(value[i]));
+      }
+      return x;
```

Comment:
While you are working on this can you also use meaningful variable names instead of x and maybe i. It'll be easier in the future if someone else is looking into the code.

Reply:
Yeah I can absolutely modify that! Normally, I'd have a lot more of these details fine-tuned by now, but I've been having some issues setting up a good testing environment. There is an issue with my package manager that is making it difficult to install all the dependencies I need to properly test the unit tests I've added. So if you see this PR go dormant for a few more days, know that I am working to fix your and Ernesto's reviews.

Thanks so much for all your help by the way! @nizamial09

Reply:
You're welcome